_topic_maps/_topic_map_osd.yml

```diff
@@ -141,6 +141,8 @@ Distros: openshift-dedicated
 Topics:
 - Name: About machine pools
   File: nodes-machinepools-about
+- Name: Managing worker nodes
+  File: rosa-managing-worker-nodes
```

Comment:
This is working on my local build, but not on netify yet.

Reply:
And it appears that it's breaking the OSD netify on this PR altogether. I will try a different approach.

Reply:
It builds locally and appears to be an issue with netify + the OSD topic map. I will link to a local build when ready to review.